archeo/style.css

```diff
@@ -113,6 +113,7 @@ body > .is-root-container,
 .edit-post-visual-editor__post-title-wrapper,
 .wp-block-group.alignfull,
 .wp-block-group.has-background,
+.wp-block-columns.alignfull.has-background,
```

Comment:
I added this rule because without it, Gutenberg's default padding for columns with a background color is applying and messing up the alignment:

Before | After
---|---
<img width="1904" alt="Screen Shot 2022-02-09 at 12 50 55 PM" src="https://user-images.githubusercontent.com/5375500/153260495-4e016840-2164-4255-8001-0b820a789826.png"> | <img width="1904" alt="Screen Shot 2022-02-09 at 12 50 40 PM" src="https://user-images.githubusercontent.com/5375500/153260524-d199016a-9900-4184-ab9f-4ffee57cb9b1.png">

Reply:
We should probably bubble this change up to TT2, Blockbase, Stewart and Livro.

Reply:
The padding in the editor is a little too intense — not sure why that doesn't match? 
<img width="361" alt="Screen Shot 2022-02-10 at 11 22 12 AM" src="https://user-images.githubusercontent.com/1202812/153450729-034e7a58-2127-4d2c-af2c-8f480f5c338c.png">

Reply:
I missed the equivalent rule for targeting the editor. I added it here: https://github.com/Automattic/themes/pull/5487/commits/74bfa700042a05b36f84ce74f474bc566fe0ec20#diff-2769b329cf3d32aab10e3f345c0da915cd1f6d5ffba70ebdeb7c5786597dca3dR119

Do you think these additions make sense? We are saying, "apply the site wide padding and negative margins for columns that are aligned full with a background color".

Reply:
> Do you think these additions make sense?

Yeah I think this makes sense, especially with the background color applied. Interested to see what anyone else thinks though, in case we're missing something.